examples/ssd1306_i2c/ssd1306_i2c.c

```diff
@@ -0,0 +1,70 @@
+/* vim: set expandtab ts=4 sw=4 :
```

Comment:
Hello fellow vim user :)
I don't think we should include configuration of our favourite editors in esp-open-rtos, as not everybody uses them and it might confuse others. It just my personal opinion.
I think it's better to include more descriptive information in the header comment or/and licensing information.